src/components/tabular-grid/column/tabular-grid-column.tsx

```diff
@@ -380,7 +389,16 @@ export class ChTabularGridColumn {
 
   render() {
     return (
-      <Host>
+      <Host
+        part={
+          this.parts &&
+          tokenMap({
+            [TABULAR_GRID_PARTS_DICTIONARY.COLUMN]: true,
+            [this.columnId]: true,
+            [this.parts.toString()]: typeof this.parts === "string"
+          })
```

Comment:
In general, we can't conditional bind attributes with `&&`, because when the expresion is `false`, it will still generate the attribute in the HTML.
In this case, when `parts === false`, `part="false"` will be reflected in the HTML.

```suggestion
          this.parts
            ? tokenMap({
              [TABULAR_GRID_PARTS_DICTIONARY.COLUMN]: true,
              [this.columnId]: true,
              [this.parts.toString()]: typeof this.parts === "string"
            })
            : null
```